assets/js/search.js

```diff
@@ -0,0 +1,218 @@
+/**
+ * Javascript specially for the IaC Library page.
+ */
+
+(function () {
+
+  // Returns a function, that, as long as it continues to be invoked, will not
+  // be triggered. The function will be called after it stops being called for
+  // N milliseconds. If `immediate` is passed, trigger the function on the
+  // leading edge, instead of the trailing.
+  // From: https://davidwalsh.name/javascript-debounce-function
+  function debounce(func, wait, immediate) {
+    let timeout;
+    return function () {
+      let context = this,
+        args = arguments;
+      let later = function () {
+        timeout = null;
+        if (!immediate) func.apply(context, args);
+      };
+      let callNow = immediate && !timeout;
+      clearTimeout(timeout);
+      timeout = setTimeout(later, wait);
+      if (callNow) func.apply(context, args);
+    };
+  }
+
+  function showItemsCount(totalCount, numSubmodules) {
+    if (searchEntry.type == 'libraryEntries') {
+      if (totalCount > 0 && numSubmodules > 0) {
+        $('#search-results-count').html("<strong>" + totalCount + "</strong> repos (~<strong>" + numSubmodules + "</strong> modules)");
```

Comment:
Can we add `<strong>` through CSS and use the least tags as possible here? Can we also just show `X found` for both library page and guides page so we can remove these ifs here?